peekingduck/configs/model/yolo.yml

```diff
@@ -2,12 +2,16 @@ input: ["img"]
 output: ["bboxes", "bbox_labels", "bbox_scores"]
 
 model_type: v4tiny # v4 or v4tiny
-classes: ../peekingduck_weights/yolov4/coco.names
-weights_dir: ["../peekingduck_weights/yolov4"]
-blob_file: "yolo.zip"
-graph_files: {
-    v4: ../peekingduck_weights/yolov4/yolov4.pb,
-    v4tiny: ../peekingduck_weights/yolov4/yolov4-tiny.pb
+
+weights_parent_dir: default
```

Comment:
Can use `null` if we are considering using `None` to signify the use of default value